### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by xm-y02**

### CODE REVIEW

**Comment by xm-y02** [comment]
@DonglinLiJCU

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+                "baker-miller pink": "#ff91af", "baby pink": "#f4c2c2",
+                "baby blue": "#89cff0", "bittersweet": "#fe6f5e",
+                "black olive": "#3b3c36", "bubble gum": "#ffc1cc",
+                "byzantine": "#bd33a4", "cadetBlue1": "#98f5ff",
+                "candy apple red": "#ff0800", "carrot orange": "#ed9121", "cerulean frost": "#6d9bc3",
+                "daffodil": "#ffff31", "dark lavender": "#734f96"}
+
+colour_name = input("Enter a colour name: ").lower()
+while colour_name != "":
+    print(f"The code for \"{colour_name}\" is {COLOUR_CODES.get(colour_name)}")
```
Comment (DonglinLiJCU) [inline prac_05/hex_colours.py:13]: Consider add one case to handle the colour_name don't exist.

File: prac_05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    try:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    except KeyError:
+        print("Invalid short state")
+    state_code = input("Enter short state: ").upper()
```
Comment (DonglinLiJCU) [inline prac_05/state_names.py:19]: There is no loop to print all the states and names as required

File: prac_05/wimbledon.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        except KeyError:
+            champion_to_count[record[INDEX_CHAMPION]] = 1
+    return champion_to_count, countries
+
+
+def display_results(champion_to_count, countries):
+    print("Wimbledon Champions: ")
+    for name, count in champion_to_count.items():
+        print(name, count)
+    print(f"\nThese {len(countries)} countries have won Wimbledon: ")
```
Comment (DonglinLiJCU) [inline prac_05/wimbledon.py:28]: Consider return the string to main, let main function print

File: prac_05/hex_colours.py (hunk 1 above)
Comment (DonglinLiJCU) [inline prac_05/hex_colours.py:13]: Consider add one case to handle the colour_name don't exist.

**Pull request merged**